alf/utils/value_ops.py

```diff
@@ -22,22 +22,21 @@
 
 
 def action_importance_ratio(action_distribution, collect_action_distribution,
-                            action, action_spec, calling_class_name, scope,
+                            action, action_spec, clipping_mode, scope,
                             importance_ratio_clipping, log_prob_clipping,
                             check_numerics, debug_summaries):
     """ ratio for importance sampling, used in PPO loss and vtrace loss.
 
-        Caller has to set tf.name_scope() and nest the call to this function
-            inside the with statement.
+        Caller has to save tf.name_scope() and pass scope to this function.
 
         Args:
             action_distribution: Probability of actions under target policy.
```

Comment:
include the type in the comment like the following
arg_name (arg_type): description